src/jdk.compiler/share/classes/com/sun/tools/javac/resources/compiler_de.properties

```diff
@@ -2286,6 +2299,8 @@ compiler.misc.feature.flexible.constructors=Flexible Konstruktoren
 
 compiler.misc.feature.module.imports=Modulimporte
 
+compiler.misc.feature.java.base.transitive=Transitiver Modifikator für java.base
```

Comment:
Fixed in https://github.com/openjdk/jdk/pull/22588/commits/c7bd52950eeb85581aa7dc2c6fbbb02a4f4cff11.
I'll check in with the translation team to ask for it not to be translated, I also left a comment for translators. I only provided the fix for _de_, since it is not obvious to me how to correct it for _ja_ or _zh_CN_.

Regarding the jshell changes in this PR, they are a result of translation fixes we filed with the translation team from previous drops. You are correct that https://github.com/openjdk/jdk/commit/bcebb0c53c1e4629ebde534e237a86c161130fff did not make it in this PR because it was too late. The localized changes for that commit will come in the RDP2 drop.

Regarding the alignment issues in the translated versions, I don't think the translation team would adapt those changes because they are aesthetic, not functional, and this spacing is likely consistent with their l10n language/punctuation rules. If I made this change, we would have to manually adjust these on our side every drop which I don't think would be ideal.